opensrp-chw/src/kk/java/org/smartregister/chw/interactor/AncHomeVisitInteractorFlv.java

```diff
@@ -969,4 +982,66 @@ public void onPayloadReceived(BaseAncHomeVisitAction baseAncHomeVisitAction) {
             Timber.v("onPayloadReceived");
         }
     }
+
+    private class ChwObservationsAction implements BaseAncHomeVisitAction.AncHomeVisitActionHelper {
+        private String anyone_presence;
+        private Context context;
+
+        @Override
+        public void onJsonFormLoaded(String s, Context context, Map<String, List<VisitDetail>> map) {
+            this.context = context;
+        }
+
+        @Override
+        public String getPreProcessed() {
+            return null;
+        }
+
+        @Override
+        public void onPayloadReceived(String jsonPayload) {
+            try {
+                JSONObject jsonObject = new JSONObject(jsonPayload);
+                anyone_presence = JsonFormUtils.getCheckBoxValue(jsonObject, "anyone_present");
+            } catch (JSONException e) {
+                Timber.e(e);
+            }
+        }
+
+        @Override
+        public BaseAncHomeVisitAction.ScheduleStatus getPreProcessedStatus() {
+            return null;
+        }
+
+        @Override
+        public String getPreProcessedSubTitle() {
+            return null;
+        }
+
+        @Override
+        public String postProcess(String s) {
+            return null;
+        }
+
+        @Override
+        public String evaluateSubTitle() {
+            return MessageFormat.format(context.getString(R.string.chw_observations_evaluate_sub_title), anyone_presence);
```

Comment:
This can cause NPE, the variable `anyone_present` needs to be initialized during declaration you can just initialize it to an empty string then here check if its empty return empty string otherwise return the formatted message

Something like this below
```suggestion
            if(anyone_presence.isEmpty()) return "";
            return MessageFormat.format(context.getString(R.string.chw_observations_evaluate_sub_title), anyone_presence);
```